File: kotlin_android_dev/app/src/main/kotlin/com/kotlin_android_dev/named_theme/LightTheme.kt
package com.kotlin_android_dev.named_theme

import androidx.compose.ui.graphics.Color

class LightTheme : BaseNamedTheme() {
    override fun backgroundPrimary(): Color {
        return Color(255,255,255)
    }

    override fun backgroundSecondary(): Color {
        return Color(245,247,250)
    }

    override fun border(): Color {
        return Color(230,234,239)
    }

    override fun brand(): Color {
        return Color(255,119,0)
    }

    override fun onBrand(): Color {
        return Color(255,255,255)
    }

    override fun text(): Color {
        return Color(34,34,34)
    }

    override fun hintText(): Color {
        return Color(160,160,160)
    }
}

File: dart_web_boilerplate/app/lib/utilities/themes/light_theme.dart
import 'package:dart_web_boilerplate/utilities/themes/base_theme.dart';
import 'package:flutter/material.dart';

@immutable
final class LightTheme extends BaseTheme {
  const LightTheme();

  @override
  Color get backgroundPrimary => Color.fromRGBO(255, 255, 255, 1.0);

  @override
  Color get backgroundSecondary => Color.fromRGBO(245, 247, 250, 1.0);

  @override
  Color get border => Color.fromRGBO(230, 234, 239, 1.0);

  @override
  Color get brand => Color.fromRGBO(255, 119, 0, 1.0);

  @override
  Color get onBrand => Color.fromRGBO(255, 255, 255, 1.0);

  @override
  Color get text => Color.fromRGBO(34, 34, 34, 1.0);

  @override
  Color get hintText => Color.fromRGBO(160, 160, 160, 1.0);
}
